contrib/ansible/roles/metrika-agent-install/templates/agent.yml.j2

```diff
@@ -50,28 +54,40 @@ runtime:
 
     # level: logging level for runtime logs.
     #
-    # Possible values are: info, warning (recommended), debug, error.
-    level: warning
-
-    # disable_fingerprint_validation: disables fingerprint validation on startup.
-    #
-    # Fingerprint validation is enabled by default and the agent will exit
-    # immediately if checksums of the newly retrieved hostname and the cached do
-    # not match. Checksum is cached under /Users/sjonpaul.brown/.cache/metrikad/fingerprint.
-    disable_fingerprint_validation: false
-
-  # metrics_addr: string, network address to get Prometheus metrics about the
-  # agent's runtime.
-  metrics_addr: 127.0.0.1:9000
+    # Possible values are: info (recommended), warning, debug, error.
+    level: info
+
+  # disable_fingerprint_validation: disables fingerprint validation on startup.
+  #
+  # Fingerprint validation is enabled by default and the agent will exit
+  # immediately if checksums of the newly retrieved hostname and the cached do
+  # not match. Checksum is cached under $HOME/.cache/metrikad/fingerprint.
+  disable_fingerprint_validation: false
+
+  # http_addr: string, network address to listen for HTTP requests to.
+  #  - Get Prometheus metrics about the agent's runtime (GET /metrics).
+  #  - Update its logging level (PUT /loglvl).
+  #
+  # Default value is empty string which disables HTTP across the agent. Enabling
+  # any agent local endpoints, requires setting this value (i.e. 127.0.0.1:9000)
```

Comment:
```suggestion
  # any agent local endpoints, requires setting this value (i.e. 127.0.0.1:9999)
```

Reply:
Thank you for the feedback @vishalchangrani. Addressed in 548b72c.